test/bats/helpers.bash

```diff
@@ -68,6 +68,7 @@ wait_for_process() {
       return 0
     else
       sleep "$sleep_time"
+      echo "# retrying $cmd" >&3
```

Comment:
curious what's FD 3?

Reply:
This the FD for the TAP stream produced by BATS. The TAP stream is always outputted to the console regardless of success or failure. I think it might be helpful to see if retries were regarding for a success and not just a failure. https://bats-core.readthedocs.io/en/stable/writing-tests.html#printing-to-the-terminal